README.md

```diff
@@ -1,38 +1,84 @@
 
-Quire is centered around [Hugo](https://github.com/gohugoio/hugo), and uses [PrinceXML](http://www.princexml.com/) for PDF generation and [pandoc](https://pandoc.org/) for EPUBs. It consists of four primary repositories, all of which but this one are closed during development.
+Quire is centered around [**Hugo**](https://github.com/gohugoio/hugo), and uses [PrinceXML](http://www.princexml.com/) for PDF generation, [pandoc](https://pandoc.org/) for EPUBs, and [Kindle Previewer](https://www.amazon.com/gp/feature.html?ie=UTF8&docId=1000765261) for MOBIs. It consists of four primary repositories, all of which but this one are private during development ([sign up for access](https://docs.google.com/forms/d/e/1FAIpQLSckvPWWyyfZJko6JTqf3slcXCV8vcCgQjAzoW4MfHEt9hDuxQ/viewform?usp=sf_link)):
 
-- [quire](https://github.com/gettypubs/quire) (documentation and homepage)
+- [quire](https://github.com/gettypubs/quire) (website and documentation)
 - [quire-cli](https://github.com/gettypubs/quire-cli)
 - [quire-starter](https://github.com/gettypubs/quire-starter)
 - [quire-starter-theme](https://github.com/gettypubs/quire-starter-theme)
 
-## License
+## Quire Contributors
+
+### Core Team
+
+- Greg Albers ([@geealbers](https://github.com/geealbers)), product manager
+- David Newbury ([@workergnome](https://github.com/workergnome)), product manager
+- Matthew Hrudka ([@mphstudios](https://github.com/mphstudios)), lead maintainer
+- Erin Cecele Dunigan ([@Erin-Cecele](https://github.com/Erin-Cecele)), community manager
+
+Contact us at [quire@getty.edu](mailto:quire@getty.edu).
+
+### Contributors
+
+Thank you to all our growing list of contributors who have made Quire what it is, and are working on what it will be! They’ve contributed code, written and edited documentation, run events, and blogged and spoken about Quire.
+
+- [@aheltonsjma](https://github.com/aheltonsjma)
+- [@anderspollack](https://github.com/anderspollack)
+- [@antoinentl](https://github.com/antoinentl)
+- [@daniel-keller](https://github.com/daniel-keller)
+- [@egardner](https://github.com/egardner)
+- [@fchasen](https://github.com/fchasen)
+- [@ggarcia0596](https://github.com/ggarcia0596)
+- [@kalvarenga](https://github.com/kalvarenga)
+- [@kristhayer11](https://github.com/kristhayer11)
+- [@mandrijauskas](https://github.com/mandrijauskas)
+- [@MillsArtMuseum](https://github.com/MillsArtMuseum)
+- [@mbelhu](https://github.com/mbelhu)
+- [@naeluh](https://github.com/naeluh)
+- [@nancyum](https://github.com/nancyum)
+- [@oncletom](https://github.com/oncletom)
+- [@swambold1](https://github.com/swambold1)
+- [@victoriabarry](https://github.com/victoriabarry)
+
+## This Repository
+
+This repository, https://github.com/gettypubs/quire, is the central hub for Quire, and is also the Quire website and documention, which is built with Quire.
+
+This repo was last built with the following software versions:
+
+- Quire 0.18.0
+- Node 12.18.3 / npm 6.14.6
```

Comment:
I like that you added software versions. I'll add this to the list of website sections that will occasionally need to be updated.